cpu/cc26x0/vendor/driverlib/setup_rom.c

```diff
@@ -363,6 +364,7 @@ SetupAfterColdResetWakeupFromShutDownCfg3( uint32_t ccfg_ModeConfReg )
                                                                    // Set XOSC_LF in bypass mode to allow external 32 kHz clock
         HWREG( AUX_DDI0_OSC_BASE + DDI_O_SET + DDI_0_OSC_O_CTL0 ) = DDI_0_OSC_CTL0_XOSC_LF_DIG_BYPASS;
         // Fall through to set XOSC_LF as SCLK_LF source
+        [[fallthrough]];

```

Comment:
my compiler does not like this

Reply:
the vendor code did not bring it so it caused a compilation error, the initialization has to fail if this condition is not met deliberately. You could use the gcc attribute` __attribute__ ((fallthrough))`. but we will need gcc(>= 7).

Reply:
`/* fall-though */` should work too